crates/examples/testfiles/elf/base.objdump

```diff
@@ -122,11 +122,11 @@ Dynamic relocations
 (200fd0, Relocation { kind: Unknown, encoding: Generic, size: 0, target: Symbol(SymbolIndex(2)), addend: 0, implicit_addend: false, flags: Elf { r_type: 7 } })
 
 Import { library: "", name: "_ITM_deregisterTMCloneTable" }
-Import { library: "", name: "printf" }
-Import { library: "", name: "__libc_start_main" }
+Import { library: "GLIBC_2.2.5", name: "printf" }
+Import { library: "GLIBC_2.2.5", name: "__libc_start_main" }
 Import { library: "", name: "__gmon_start__" }
 Import { library: "", name: "_ITM_registerTMCloneTable" }
-Import { library: "", name: "__cxa_finalize" }
+Import { library: "GLIBC_2.2.5", name: "__cxa_finalize" }
```

Comment:
The version info for this file looks like:
```
    VersionNeed {
        Version: 1
        AuxCount: 1
        Filename: "libc.so.6" (0x1)
        AuxOffset: 16
        NextOffset: 0
        Aux {
            Hash: 0x9691A75
            Flags: 0x0
            Index: 2
            Name: "GLIBC_2.2.5" (0x33)
            NextOffset: 0
        }
    }
```

I would have expected the `name` for these to be `libc.so.6`.